crxmake.sh

```diff
@@ -59,9 +62,16 @@ sig_len_hex=$(byte_swap $(printf '%08x\n' $(ls -l "$sig" | awk '{print $5}')))
 
 echo "Wrote $crx"
 
+# move crx to artifacts folder in circleci
+if [ $CIRCLE_ARTIFACTS ]; then
```

Comment:
@igor-ka I spent a whole 10min investigating (and no dice), maybe I can spend a little bit more time?